microshift_running_apps/microshift-deleting-resource-manifests.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-deleting-resource-manifests"]
+= Deleting resource manifests
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-deleting-resource-manifests
+
+toc::[]
+
+{microshift-short} supports the deletion of resource manifests for the following use cases:
+
+* Manifest removal - when you need to completely remove a resource from the cluster.
+* Manifest upgrade - during an application upgrade, some resources may need to be removed while others are retained to preserve data.
+
+when creating new manifests, you can use resource manifest deletion to remove or update old objects, ensuring there are no conflicts or issues.
+
+[IMPORTANT]
+====
+Manifest files placed in the `delete` subdirectories are not automatically removed and require manual deletion.
```

Comment:
I don't understand the note, because the procedures seem to be telling the user to put manifests into the delete subdirectories and then upon restart, they will automatically be deleted. Maybe I am misunderstanding something.

Reply:
meaning there are more steps involved in deleting the manifest file after they are placed in the delete subdirectory. for example see step 2,3 and 4 in the "Manifest removal scenario for RPM systems"

Reply:
```suggestion
Manifest files placed in the `delete` subdirectories are not always automatically removed. In some cases the manifests require manual deletion, as described in the following sections. 
```